src/evm/initialize-provider.ts

```diff
@@ -35,14 +36,14 @@ export const initializeProvider = async (
   const voidSigner = new ethers.VoidSigner(ethers.constants.AddressZero, provider);
 
   // Fetch current block number
-  const [errorGetBlockNumber, currentBlock] = await retryGo(() => provider.getBlockNumber());
-  if (errorGetBlockNumber || isNil(currentBlock)) {
+  const currentBlock = await go(() => provider.getBlockNumber(), { timeoutMs: DEFAULT_RETRY_TIMEOUT_MS });
+  if (!currentBlock.success) {
     const message = 'Failed to fetch the blockNumber';
-    const log = node.logger.pend('ERROR', message, errorGetBlockNumber);
+    const log = node.logger.pend('ERROR', message, currentBlock.error);
     return [[log], null];
   }
-  const currentBlockMessage = `Current block number for chainId ${chain.id}: ${currentBlock}`;
-  const currentBlockLog = node.logger.pend('INFO', currentBlockMessage);
+  const currentBlockMessage = `Current block number for chainId ${chain.id}: ${currentBlock.data}`;
+  const currentBlockLog = node.logger.pend('DEBUG', currentBlockMessage);
```

Comment:
I was asked to make this INFO [here](https://github.com/api3dao/airkeeper/pull/31#discussion_r825887889). Same with gasTarget.

Reply:
:+1:  Thanks for catching this... fixed